code/game/gamemodes/technomancer/spells/audible_deception.dm

```diff
@@ -34,18 +34,18 @@
 		"Gun Firing"			=	'sound/weapons/gunshot.ogg',
 		"Desert Eagle Firing"	=	'sound/weapons/deagle.ogg',
 		"Rifle Firing"			=	'sound/weapons/rifleshot.ogg',
-		"Rifle Firing 2"		=	'sound/weapons/svd_shot.ogg',
-		"Sniper Firing"			=	'sound/weapons/sniper.ogg',
+		"Sniper Rifle Firing"	=	'sound/weapons/svd_shot.ogg',
+		"AT Rifle Firing"		=	'sound/weapons/sniper.ogg',
 		"Shotgun Firing"		=	'sound/weapons/shotgun.ogg',
-		"Semi-automatic Firing"	=	'sound/weapons/semiauto.ogg',
+		"Handgun Firing"		=	'sound/weapons/gunshot3.ogg',
 		"Machinegun Firing"		=	'sound/weapons/machinegun.ogg',
 		"Rocket Launcher Firing"=	'sound/weapons/rpg.ogg',
 		"Taser Firing"			=	'sound/weapons/Taser.ogg',
 		"Laser Gun Firing"		=	'sound/weapons/laser.ogg',
 		"E-Luger Firing"		=	'sound/weapons/eLuger.ogg',
 		"Xray Gun Firing"		=	'sound/weapons/laser3.ogg',
 		"Pulse Gun Firing"		=	'sound/weapons/pulse.ogg',
-		"Gauss Gun Firing"		=	'sound/weapons/gauss_shoot.ogg',
+		"Energy Sniper Firing"	=	'sound/weapons/gauss_shoot.ogg',
```

Comment:
Same here.

Reply:
That's just Github making it look stupid.